cloudvolume/datasource/precomputed/spatial_index.py

```diff
@@ -209,16 +209,26 @@ def fetch_index_files(self, index_files, progress=None):
 
     return { res['filename']: res['content'] for res in results }
 
-  def fetch_all_index_files(self, allow_missing=False, progress=None):
+  def fetch_all_index_files(self, allow_missing=False, progress=None, parallel=1):
     """Generator returning batches of (filename, json)"""
```

Comment:
Might be good to note in the function docstring or in a comment that parallel doesn't directly activate a thread or process but only affects the chunking in this function. Usually, the convention in my code is that parallel eventually ends up spinning up a thread at the "leaf" level of the call tree. This usage makes sense, but is a little exceptional.